soh/soh/Enhancements/TimeSavers/SkipCutscene/SkipIntro.cpp

```diff
@@ -12,7 +12,7 @@ extern "C" {
 void SkipIntro_Register() {
     REGISTER_VB_SHOULD(VB_PLAY_TRANSITION_CS, {
         if (CVarGetInteger(CVAR_ENHANCEMENT("TimeSavers.SkipCutscene.Intro"), IS_RANDO)) {
-            if (gSaveContext.entranceIndex == ENTR_LINKS_HOUSE_0 && gSaveContext.cutsceneIndex == 0xFFF1) {
+            if (gSaveContext.entranceIndex == IS_RANDO ? Entrance_GetOverride(ENTR_LINKS_HOUSE_0) : ENTR_LINKS_HOUSE_0 && gSaveContext.cutsceneIndex == 0xFFF1) {
```

Comment:
I don't think this is correct as is. You need parenthesis to contain the ternary, otherwise the `gSaveContext.entranceIndex == IS_RANDO` is being considered the value for the ternary condition.
```suggestion
            if (gSaveContext.entranceIndex == (IS_RANDO ? Entrance_GetOverride(ENTR_LINKS_HOUSE_0) : ENTR_LINKS_HOUSE_0) && gSaveContext.cutsceneIndex == 0xFFF1) {
```

Reply:
Ah good catch, the testing I had done hadn't yet caught that it was just always skipping the cutscene in rando.